modules/monitoring-creating-user-defined-alerting-rules.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="creating-user-defined-alerting-rules_{context}"]
+= Creating user-defined alerting rules
+
+As a cluster administrator, you can create user-defined alerting rules based on platform metrics. 
+These alerting rules will fire alerts based on the values of chosen metrics.
+
+[NOTE]
+====
+If you create a new, customized `AlertingRule` resource based on an existing platform alerting rule, you should silence the original alert to avoid receiving conflicting alerts.
+====
+
+.Prerequisites
+
+* You are logged in as a user that has the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
```

Comment:
same as https://github.com/openshift/openshift-docs/pull/48754/files#r939802522
we should add
You have enabled TechPreview feature and wait for all the nodes become ready

Reply:
Added a line to the prerequisites